frontend/front/src/pages/Surveyor/Components/SurveyorSurvey.js

```diff
@@ -1,7 +1,7 @@
 import React, { forwardRef } from "react";
 import SurveyComponent from "../../../components/SurveyComponent/SurveyComponent";
 
-export const SURVEYOR_SURVEY_ID = "2";
+export const SURVEYOR_SURVEY_ID = "1";
```

Comment:
Here's what I'm referring to in the initial comment on this PR. Should this be a constant or is it meant to be dynamic? Why was it set to '2' before when only survey 1 contains any questions?

Reply:
Ah okay, I think this got missed somehow. The original idea was that there was going to be different versions of the survey for the public/surveyor use cases but we've moved away from that I guess, and as you point out there's only one survey in the actual DB. But it is meant to be hard-coded with the survey ID based on the use-case.